MetaMask/metamask-extension
# UX: Remove AccountPicker hover styling when disabled

## **Description**
This ensures the AccountPicker background doesn't change background upon hover when the component is disabled.


## **Manual testing steps**

1.  Hover over the AccountPicker -- see background change
2. Click on "Swap" button
3. Hover over the disabled AccountPicker -- see background doesn't change

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
